.github/workflows/review.yml

```diff
@@ -99,9 +99,8 @@ jobs:
       fetch_cmd_aarch64-darwin: ${{ steps.report.outputs.fetch_cmd_aarch64-darwin }}
 
     steps:
-      - name: prepare /nix
-        run: sudo mkdir /mnt/nix && sudo mount -m -o bind /mnt/nix /nix
-        if: ${{ matrix.system == 'x86_64-linux' || matrix.system == 'aarch64-linux' }}
```

Comment:
Should the old impl be kept on non-linux runners?

EDIT: I'm blind: the old impl was also only used on linux!

Reply:
This begs the question, is there a similar solution available for reclaiming disk space on darwin runners?

Definitely not in scope here.